ompi/mca/osc/ucx/osc_ucx_component.c

```diff
@@ -102,8 +104,15 @@ static int component_open(void) {
 }
 
 static int component_register(void) {
+    unsigned major          = 0;
+    unsigned minor          = 0;
+    unsigned release_number = 0;
     char *description_str;
-    mca_osc_ucx_component.priority = 0;
+
+    ucp_get_version(&major, &minor, &release_number);
+
+    mca_osc_ucx_component.priority = UCX_VERSION(major, minor, release_number) >= UCX_VERSION(1, 5, 0) ? 200 : 0;
```

Comment:
We need to discuss this level.

Reply:
you mean the '200' priority number, or the version check?

Reply:
'200' used to allow add more OSC implementations between RDMA & UCX

Reply:
The 200 level. You are changing behavior on more than just Mellanox platforms. It should be no higher than 100. We will discuss after the holiday.

Reply:
Lets discuss on Nov 27 web-ex call.